### PR TITLE
72 new bootstrap template

### PR DESCRIPTION
**Pull request by simplews**
- [ ] @Kmalynss 
- [ ] @aadmin 
- [ ] @mis6ko13 

Changed theme to [proposed](https://bootstrapious.com/p/universal-business-e-commerce-template):
- Home page, issues pages, issues list page, user list page redesigned
- New css aplied
- Fixed some minor bugs with views after new css added
- Changed names of color variables (may be changed soon to something more semantic) 
- Changed favicon set

### CODE REVIEW

File: app/assets/javascripts/front.js [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+    animations();
+    counters();
+
+});
+
+/* menu sliding */
+
+function menuSliding() {
+
+
```
Comment (f1nwe) [inline app/assets/javascripts/front.js]: Remove empty lines.

File: app/assets/javascripts/front.js [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+    );
+    $('.dropdown').on('hide.bs.dropdown', function (e) {
+  if ($(window).width() > 750) {
+      $(this).find('.dropdown-menu').first().stop(true, true).slideUp();
+  }
+  else {
+      $(this).find('.dropdown-menu').first().stop(true, true).hide();
+  }
+    });
+
```
Comment (f1nwe) [inline app/assets/javascripts/front.js]: Remove empty lines in whole file. Correct style of code and indents.

File: app/views/issues/_issue.html.slim [hunk 3]
```diff
@@ -1,16 +1,22 @@
...
-      .timestamp
-        span.author Автор: #{issue.user.full_name} |
-        span  Додано:    #{time_ago_in_words(issue.created_at)} тому
-        br
-        span  Локація: #{issue.location}
+      .clearfix
+        p.author-category
+          span.author Автор: #{issue.user.full_name }
+          br
+          span  #{'Категорія: '}
```
Comment (f1nwe) [inline app/views/issues/_issue.html.slim]: Why are you using interpolation here?

File: app/views/issues/index.html.slim [hunk 4]
```diff
@@ -1,15 +1,24 @@
...
+  .container
+    .row
+      #blog-listing-medium.col-md-9
+        .pull-right
+          .dropdown.issue-sort-by
+            a.dropdown-toggle href='#' data-toggle="dropdown"
+              span Сортувати
+              span.caret
+              ul.dropdown-menu role='menu'
+                li= link_to 'Спочатку нові', '#'
```
Comment (f1nwe) [inline app/views/issues/index.html.slim]: Maybe it is better to sort via categories?

File: app/views/layouts/_logged_content.html.slim [hunk 5]
```diff
@@ -1,22 +1,31 @@
...
+      a.dropdown-toggle[href='#' data-toggle="dropdown"]
+        | Меню користувача
+        b.caret
       ul.dropdown-menu role='menu'
         li
-          = link_to 'Мої звернення', user_path(current_user)
+          img.user_img src='#{current_user.image_url}'
+          = current_user.first_name
+        li
+          = link_to 'Мої звернення', '#'
```
Comment (f1nwe) [inline app/views/layouts/_logged_content.html.slim]: We already have this link.

File: app/views/layouts/_logged_content.html.slim [hunk 6]
```diff
@@ -1,22 +1,31 @@
...
+        li
+          = link_to 'Всі звернення', issues_path
         li
-          = link_to('Вийти', destroy_user_session_path, method: :delete)
\ No newline at end of file
+          = link_to('Вийти', destroy_user_session_path, :method => :delete)
```
Comment (f1nwe) [inline app/views/layouts/_logged_content.html.slim]: Use new ruby syntax.

File: app/views/issues/_issue.html.slim (hunk 3 above)
Comment (simplews) [inline app/views/issues/_issue.html.slim]: This was added by @mis6ko13 , so I didn't change it this time.

File: app/views/issues/index.html.slim (hunk 4 above)
Comment (simplews) [inline app/views/issues/index.html.slim]: This was also added by @mis6ko13 in his PR, so I didn't change it this time.
Sort via categories, i think, we can add in addition to this. We have empty sidebar on the right according to chosen template. So we can use it, for example to add categories filter etc/

File: app/views/layouts/_logged_content.html.slim (hunk 5 above)
Comment (simplews) [inline app/views/layouts/_logged_content.html.slim]: Fixed, will be in next commit

File: app/views/layouts/_logged_content.html.slim (hunk 6 above)
Comment (simplews) [inline app/views/layouts/_logged_content.html.slim]: Fixed, will be in next commit

File: app/views/issues/_issue.html.slim (hunk 3 above)
Comment (mis6ko13) [inline app/views/issues/_issue.html.slim]: Use  - @issues.each do |issue| at the top of the partial, as you are receiving an @issues collection from  = render 'issue', collection: @issues in index.html.slim

File: app/views/layouts/_logged_content.html.slim [hunk 7]
```diff
@@ -1,22 +1,31 @@
 - if current_user.banned?
   div.banned.bg-danger Ваш обліковий запис заблоковано
-ul.navbar-nav.nav.pull-right
-  - if current_user.admin? || current_user.moderator?
-      li.nav-text-link
+- if current_user.admin? || current_user.moderator?
+  li.dropdown
+    a.dropdown-toggle[href="#" data-toggle="dropdown"]
+      | Меню модератора
```
Comment (mrkava) [inline app/views/layouts/_logged_content.html.slim]: Moderators and admins also can send issues and watch their profile, so we need links for this in top menu not only for reporters but also for admins and moders

Comment (simplews) [inline app/views/layouts/_logged_content.html.slim]: Ok, I got it

File: app/assets/javascripts/front.js [hunk 8]
```diff
@@ -0,0 +1,114 @@
...
+    }, function () {
+  $(this).removeClass('animated');
+  $(this).removeClass($(this).data('animate-hover'));
+    });
+}
+
+function animationsSlider() {
+    var delayTimeSlider = 400;
+    $('.owl-item:not(.active) [data-animate-always]').each(function () {
+  $(this).removeClass('animated');
```
Comment (igorkasyanchuk) [inline app/assets/javascripts/front.js]: formatting looks bad, add indentation

Comment (simplews) [inline app/assets/javascripts/front.js]: Fixed

**Pull request merged**